### ISSUE TITLE
PropertyChange for TimePicker in CLOCK mode not working on Android 5 Lolipop

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/NativeScript#2940]
Dependency observable properties **hour** and **minute** are not working under Android 5 (working for Android 4.4 and Android 6 & 7 thou)
Code to reproduce [here](https://github.com/NickIliev/NativeScript-issues/tree/master/NativeScript/issue_2901_tsc) or:

**_AndroidManifext.xml_**

```
<!-- Default style for TimePicker - in spinner mode -->
<style name="SpinnerTimePicker" parent="android:Widget.Material.Light.TimePicker">
    <item name="android:timePickerMode">clock</item>
</style>
```

**_page.xml_**

```
<TimePicker id="tp" hour="{{ hour }}" minute="{{ minute }}" propertyChange="onChange"></TimePicker>
```

**_page.ts_**

```
export function onCheckChange(args:PropertyChangeData) {
    console.log(args.eventName);
    console.log(args.object);
    console.log(args.propertyName);
    console.log(args.value);
}
```

Related to [https://github.com/NativeScript/NativeScript/issues/2901](https://github.com/NativeScript/NativeScript/issues/2901)

**Comment by hamorphis** [comment]
http://stackoverflow.com/questions/38474120/timepicker-not-call-ontimechanged-in-api-21

**Comment by hamorphis** [comment]
https://code.google.com/p/android/issues/detail?id=78861

**Comment by hamorphis** [comment]
This issue is fixed with Android version 5.1 (API Level 22). The only broken version seems to be Android 5.0 (API Level 21). So we can't really fix anything in Nativescirpt. The bad news is that you cannot change the time picker mode to `spinner` programmatically: http://stackoverflow.com/questions/27277338/android-how-to-programmatically-change-timepicker-mode

**Comment by odubuc** [comment]
Alright thanks I didn't think looking for issues in Android itself... (sorry about that)
In that case I'll revert to spinner, there is too much 5.0 in use.
